api/SVGFECompositeElement.json

```diff
@@ -57,7 +57,8 @@
               "version_added": "18"
```

Comment:
I just noticed that a few lines above this, that this feature is erroneously named `in1` instead of `in`.